frontend/.prettierignore

```diff
@@ -1 +1,2 @@
 src/generated
+src/mlmd/generated
```

Comment:
nit: as mentioned in high level comment, suggest move to frontend/third_party/mlmd

Reply:
I tried the suggested path, but it looks like that `create-react-app` has restricted import from outside `src` directory. https://github.com/facebook/create-react-app/issues/1699

It requires relative path to import `mlmd` if we place the generated MLMD proto outside `src`. So I suggests to move this folder to `frontend/src/third_party/mlmd`.